templates/docker-compose.yaml.j2

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+services:
+    # web:
+    #   image: nginx:latest
+    #   ports:
+    #     - "8080:80"
+    #   volumes:
+    #     - /opt/nginx/html:/usr/share/nginx/html:ro
+    #     - /opt/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+  cache:
+    image: nginx:latest
```

Comment:
Can you pin to a tag/sha? i think the format is docker pull <image_name>@sha256:<digest>